Nos/Service/RelayService.swift

```diff
@@ -414,3 +414,28 @@ extension RelayService: WebSocketDelegate {
         }
     }
 }
+
+// MARK: NIP-05 Support
+extension RelayService {
+    func verifyInternetIdentifier(identifier: String, userPublicKey: String) async -> Bool {
```

Comment:
I wasn't sure exactly where to put this method, but since the ProfileHeader already has a reference to RelayService, it seems like a good place to put it.

Reply:
LGTM!